### DISCUSSION TITLE
Possible to delay alert from being sent once it gets a query hit?

### DISCUSSION THREAD

**Discussion by roman-tasi** [jertel/elastalert2#1201] [Q&A] [answered]
Just wondering if there is a setting that can accomplish this. For example instead of sending immediately, have it wait 10 seconds.

**Comment by jertel** [comment, marked as answer]
There's a [query_delay](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#query-delay) param you can use to time shift the queries by 10 seconds. This isn't the same as querying, detecting the alert and then waiting for 10 seconds before sending the alert, but it may achieve the same effect you're after.
